src/Nethermind/Nethermind.Merge.Plugin/MergePlugin.cs

```diff
@@ -307,7 +309,8 @@ IBlockImprovementContextFactory CreateBlockImprovementContextFactory()
                     beaconSync,
                     _api.LogManager,
                     TimeSpan.FromSeconds(mergeConfig.NewPayloadTimeout),
-                    _api.Config<IReceiptConfig>().StoreReceipts),
+                    _api.Config<IReceiptConfig>().StoreReceipts,
+                    simulateBlockProduction: _api.Config<IMergeConfig>().SimulateBlockProduction),
```

Comment:
resolve to a local variable - multiple resolves

Reply:
Done